pkg/trace/config/config.go

```diff
@@ -253,6 +252,39 @@ type DebuggerProxyConfig struct {
 	APIKey string
 }
 
+// Default AppSec configuration values
+const (
+	DefaultAppSecEnabled        = true
+	DefaultAppSecDDUrl          = ""
+	DefaultAppSecMaxPayloadSize = 5 * 1024 * 1024
+
+	DefaultAppSecObfuscationKeyRegexp   = `(?i)(p(ass)?w(((or)?d))|(phrase))|(secret)|(authorization)|(api_?key)|((access_?)?token)`
```

Comment:
I have a suggestion to improve the regex
```suggestion
	DefaultAppSecObfuscationKeyRegexp   = `(?i)(p(ass)?w(or)?d|pass$|pass(_?phrase)|secret|(api_?|private_?|public_?)key)|token|consumer_?(id|key|secret)|sign(ed|ature)`
```

It catches the following test cases 
```
password
passwd
pwd
pass
pass_phrase
passPhrase
secret
api_key
apikey
secret_key
secretkey
private_key
privatekey
public_key
publickey
token
api_token
apiToken

consumerId
consumer_ID
consumer_key
consumerKey
consumer_secret
consumerSecret

signature
signed
```

Reply:
Thanks Eslam - I slightly adapted it and added `bearer` to it in https://github.com/DataDog/datadog-agent/pull/11324/commits/c308dfbbb0d1f564c73cb1a3354ea8eac49f565b